src/Repositories/ItemRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using DomainModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Repositories
+{
+    public class ItemRepository : IItemRepository
+    {
+        private readonly DataContext _dataContext;
+
+        public ItemRepository(DataContext dataContext)
+        {
+            _dataContext = dataContext ?? throw new ArgumentNullException(nameof(dataContext));
+        }
+
+        public IQueryable<Item> All()
+        {
+            return _dataContext.Items.AsQueryable();
+        }
+
+        public void Save() { }
```

Comment:
Please implement the method. Since Save is designed to be used by both Add and Update, it should handle both scenarios. DataContext.Items is where the item should be added or updated.

Reply:
Added implementation for Save() method